sample.regexp

```diff
@@ -0,0 +1,34 @@
+\k<n>     \k'n'                                   [[:^word]]   ASCII-range         Full-range        Backslash
+\k<-n>    \k'-n'                                  ————————————————————————————————————————————————————————————
```

Comment:
You've used em dashes instead of ASCII hyphens by mistake. You might want to `s/\u2014/\u002D/g`. 😉

Reply:
Oops :) I'll fix.

Reply:
Done.

Reply:
The "continuous line" is actually how my editor renders a sequence of contiguous hyphens. 😉 I can see how it might have caused confusion, though.